src/pushstate-anchor.js

```diff
@@ -6,13 +6,14 @@
   var HTMLPushStateAnchorElement = Object.create(HTMLAnchorElement.prototype);
 
   function pushStateAnchorEventListener(event) {
-    // open in new tab
-    if (event.ctrlKey || event.metaKey || event.which === 2) {
+    var href = this.getAttribute('href');
+    // open in new tab or cross-origin requests
+    if (event.ctrlKey || event.metaKey || event.which === 2 || href.startsWith("http") && window.location.host !== new URL(href).host) {
```

Comment:
this all looks good, but can you split this into 2 separate if checks for clarity around the comments. have the first one check for opening in a new tab and the second one can check if the host is on a different domain.